pkg/JuliaInterface/read.g

```diff
@@ -12,21 +12,31 @@ ReadPackage( "JuliaInterface", "gap/utils.gi");
 ReadPackage( "JuliaInterface", "gap/helpstring.g");
 ReadPackage( "JuliaInterface", "gap/juliahelp.g");
 
-# HACK: we need to set up the GAP package override system as early as possible.
-# Since JuliaInterface is loaded before all other packages, putting it here
-# ensures that
-BindGlobal("DirectoriesPackageProgramsOverrides", rec());
-
-BindGlobal("_DirectoriesPackageProgramsOriginal", DirectoriesPackagePrograms);
 MakeReadWriteGlobal("DirectoriesPackagePrograms");
 DirectoriesPackagePrograms := function(name)
```

Comment:
This could also use `ReplaceBinding` from PR #1085 once that is merged.

In any case, I think I should add a comment above this code to indicate why it is there and point to the other parts of it (in `src/GAP_pkg.jl`).

Reply:
also the comment could indicate that most of the code below is copied from GAP's `DirectoriesPackagePrograms`

Reply:
I still think this comment and usage of ReplaceBinding would be great